src/quantum/tests/GlobalVariable.test.ts

```diff
@@ -140,4 +194,29 @@ export class GlobalVariableTEst {
         }));
     }
 
+    public "Should not replace require statement 3"() {
+        return FuseTestEnv.create(
+            {
+                project: {
+                    files: {
+                        "index.ts": `
+                        const require = 2
+                        (function(){
+                            (function(){
+                                require('test')
+                            }())
+                        }())
+                    `
+                    },
+                    plugins: [QuantumPlugin({
+                        target: "browser"
+                    })]
+                }
+            }
+        ).simple().then(test => test.browser((window, env) => {
+            const app = env.getScript("app.js");
+            app.shouldNotFindString("require('test')");
+        }));
+    }
+
 }
```

Comment:
Let's add one more
```
const require = function(str){ return str };
const hello = {
	foo  : {
		bar : {
			woo : () => return require("foobar")
			
		}
	}
}
	

module.exports = hello.foo.bar.woo()

```

and assertion

```
 const res = window.$fsx.r(0);
should(res).equal("foobar")
```

Reply:
this test fails in a weird way, it's out of the scope of what i'm fixing...